Logger/Logger.csproj

```diff
@@ -2,6 +2,14 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <CodeAnalysisTreatWarningsAsErrors>false</CodeAnalysisTreatWarningsAsErrors>
+    <LangVersion>9.0</LangVersion>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
Typically these conditions get added when you edit the project using the VS project properties GUI. In general you don't want this condition, and you want `TreatWarningsAsErrors` inside of the default PropertyGroup above

Reply:
This is where VS added everything when I set them from the properties window but I'll move them to the property file referenced in your other comment.